backend/templates/bokeh.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
https://github.com/SkywardAI/chat-frontend/blob/main/pages/ui-components/Analytics.vue Here is the frontend code, I think we return a "<div></div>" would be better